showcase/python/nox.py

```diff
@@ -36,7 +36,7 @@ def showcase(session, py):
     session.virtualenv_dirname = 'showcase-' + py
 
     # Install all test dependencies, then install this package in-place.
-    session.install('pytest')
+    session.install('pytest', 'mock')
```

Comment:
Do we need `mock` on 3.x?

Reply:
Yes, because we import it as `mock` everywhere, rather than `unittest.mock`, for compatibility with 2.7.